libs/content/workspace/src/workspace.rs

```diff
@@ -786,14 +778,12 @@ impl Workspace {
                         match result {
                             Ok((content, hmac, time_saved, seq)) => {
                                 tab.last_saved = time_saved;
-                                if hmac.is_some() {
```

Comment:
On master, the editor would find out that the content had been saved (update to `md.editor.hmac` and call to `md.editor.buffer.saved()`) only if `hmac.is_some()`. On the initial write, `hmac` is `None`, and the editor is unaware that content has been saved. On the next load, the editor proceeds as if the version last agreed upon by editor and disk is the initial empty file contents, and therefore that the loaded content is a result of a concurrent edit. This triggers the conflict resolution logic when it shouldn't have been triggered at all.

The reason for the reload is unclear but it's known not to be a safe-write failure/re-read required. It's also unclear why this is more reproducible for Parth than for Travis, with Parth reproducing sometimes on the first attempt and Travis needing sometimes over 200 attempts.